Dockerfile

```diff
@@ -34,6 +34,8 @@ ARG ONTPUB_FAMILY
 ARG ONTPUB_SPEC_HOST
 ARG DEV_SPEC
 ARG PROD_SPEC
+ARG HYGIENE_SPEC_WARN
```

Comment:
The default values for both parameters should be set to false, i.e. by default we do not run consistency checks.

Reply:
See [comment](https://github.com/edmcouncil/ontology-publisher/pull/93/files#r861933700) and [comment](https://github.com/edmcouncil/ontology-publisher/pull/93/files#r861934113)